tests/publiccloud/run_ltp.pm

```diff
@@ -156,34 +149,34 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    # LTP command line preparation
-    # The python3-paramiko is too old (2.4 on 15-SP6)
-    # The python311-paramiko is from SLE-Module-Python3-15-SP5-Updates which we have in PC tools image
-    zypper_call("in python311-paramiko python311-scp");
+    assert_script_run("cd kirk");
+    assert_script_run("python3.11 -m venv env311");
+    assert_script_run("source env311/bin/activate");
+    assert_script_run("pip3.11 install asyncssh msgpack");
 
     my $sut = ':user=' . $instance->username;
     $sut .= ':sudo=1';
     $sut .= ':key_file=$(realpath ' . $instance->provider->ssh_key . ')';
     $sut .= ':host=' . $instance->public_ip;
-    $sut .= ':reset_command=\'' . $reset_cmd . '\'';
+    $sut .= ':reset_cmd=\'' . $reset_cmd . '\'';
```

Comment:
Sorry for asking, did you noted this parameter name changed here for kirk ?

Reply:
yes, this was pointed out by author of Kirk